src/components/OfflineWithFeedback.js

```diff
@@ -45,6 +45,9 @@ const propTypes = {
     /** Additional style object for the error row */
     errorRowStyles: stylePropTypes,
 
+    /** Whether we should allow the user to close the error message */
+    isCloseable: PropTypes.bool,
+
```

Comment:
Did you add this just because we didn't have an X icon in the design? Now that I think about it I think we might want to keep the X, just so the user can hide it (and kill that RBR) if they don't care about fixing their profile picture that moment - what do you think?

Reply:
Yeah I totally got rid of it because there wasn't any X in the design, don't mind adding it back since it simplifies things!